pkg/actors/actors.go

```diff
@@ -40,19 +40,19 @@ var log = logger.NewLogger("dapr.runtime.actor")
 
 // Actors allow calling into virtual actors as well as actor state management
 type Actors interface {
-	Call(req *CallRequest) (*CallResponse, error)
+	Call(ctx context.Context, req *CallRequest) (*CallResponse, error)
```

Comment:
The PR is passing context to the methods but its not used in any meaningful way (for actual logging with a correlation and cancellation), IMO these changes are not very big and would make sense to make these changes when we are actually implementing the desired functionality.

Reply:
I agree, pass a context only where used.